src/components/SocialMedia/index.tsx

```diff
@@ -0,0 +1,25 @@
+import { FaDiscord, FaGithub, FaLinkedin } from "react-icons/fa";
+import { AiOutlineYoutube } from "react-icons/ai";
+
+export default function SocialMedia() {
+  return (
+    <div className="gap-4 flex">
+      <FaDiscord
```

Comment:
The components structure is looking great! Just a quick note – for the icons, could we wrap each of them in `Link` tags with a simple "/" as the href? Thanks! 🚀